e2e/run.sh

```diff
@@ -3,6 +3,5 @@
 # SPDX-FileCopyrightText: The RamenDR authors
 # SPDX-License-Identifier: Apache-2.0
 
-echo "Running tests..."
-
-go test -timeout 0 -v "$@"
+go test -c -o e2e
+./e2e -test.timeout 0 -test.v "$@" 2>&1 | tee e2e.log
```

Comment:
When running in the shell this works without redirecting stderr, but when running in the script the logs are not duplicated to the log file without the didirection.